templates/index.html

```diff
@@ -25,111 +25,147 @@
 <main id="main-content" class="u-no-margin--top">
   <div class="p-strip--light">
```

Comment:
We can add `is-deep` here because it's the main strip of the page. Also add `is-bordered` to have a bottom border